public_html/wp-content/mu-plugins/blocks/assets/blocks.min.css

```diff
@@ -0,0 +1,106 @@
+.wordcamp-clearfix:after {
```

Comment:
I'd love to have the build files ignored, but first we'd need some kind of post-push or pre-deploy step that would run the build and commit the changes files to the repo. I'll make a note of that as a future devex improvement, but I don't think it's a high priority for now, unless @coreymckrill or @vedanshujain see a quick and easy way to do it.

Reply:
Agreed. Let's wait until our VCS situation stabilizes.

Reply:
It's pretty annoying to have the build files always dirty while trying to switch branches, and doing `git diff` without qualifying specific files, etc.

I'm wondering if it'd be simple if, for now, we just have them ignored by Git, but not by SVN? Then when we start committing to Git directly, we can setup the pre-deploy hook?

Reply:
I tried to ignore the build files, but I couldn't get it to work, I'm assuming because the Git repo is essentially a mirror of the SVN repo, and the files are already committed there.

Reply:
I did some digging and found [`update-index --assume-unchanged`](https://stackoverflow.com/questions/10755655/git-ignore-tracked-files), which ignores them, but I think that'll also prevent us from committing the changed files to SVN. 

`git update-index --assume-unchanged public_html/wp-content/mu-plugins/blocks/assets/blocks.min.js`

At least, it will if we're using git-svn. Maybe we can have an SVN checkout of the `assets` folder just for committing those files, though? Any better ideas?

Reply:
Doh, nevermind, that still throws up an error when changing branches :(

Hopefully we'll do the full migration in the next few weeks, though, so maybe it's not a big deal.